### PR TITLE
Configure persistence from yaml file + set headers to persistence api

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

File: src/app/index.ts [hunk 1]
```diff
@@ -32,3 +33,5 @@ export { ArlasToolKitModule } from './app.module';
 export { ArlasToolkitSharedModule } from './shared.module';
 export { ArlasDataSource } from './tools/arlasDataSource';
 export { ArlasTaggerModule } from './tagger.module';
+export { AuthentSetting } from './services/authentification/authentification.service';
+export { PersitenceSetting } from './services/persistence/persistence.service';
```
Comment (alainbodiguel) [inline src/app/index.ts:37]: PersiStenceSetting ?

File: src/app/services/startup/startup.service.ts [hunk 2]
```diff
@@ -340,46 +335,29 @@ export class ArlasStartupService {
...
-                      const promises = new Array<Promise<any>>();
-                      configData.extraConfigs.forEach(extraConfig => promises.push(this.loadExtraConfig(extraConfig, configData)));
-                      return Promise.all(promises).then(() => configData);
-                  } else {
-                      return Promise.resolve(configData);
-                  }
-              })).toPromise();
-        }
-        return configDataPromise.then(configObject => this.validateConfiguration(configObject));
+        const usePersistance = (settings && settings.persistence && settings.persistence.url && settings.persistence.url !== '');
```
Comment (alainbodiguel) [inline src/app/services/startup/startup.service.ts:339]: usePersistence ?

Comment (MohamedHamouGisaia) [reply src/app/services/startup/startup.service.ts]: fixed

File: src/app/index.ts (hunk 1 above)
Comment (MohamedHamouGisaia) [reply src/app/index.ts]: fixed

**Pull request merged**